client/src/app/components/CreateProject/Form.js

```diff
@@ -18,9 +18,15 @@ class Form extends React.Component {
 
   onUpdateSelection(buttonTitle, type) {
     if (type === "stack") {
-      this.setState(prevState => ({
-        stack: [...prevState.stack, buttonTitle]
-      }));
+      if (this.state.stack.indexOf(buttonTitle) === -1 && this.state.stack.length < 5) {
```

Comment:
this.state.state is repeated 2 times in the code. It should have been destructured in a variable on top of the method. 

Please note that indexOf can be replaced by the ES6 includes, which reads better.

Even better, you could create a easy to understand variable to use in your if condition. Such as isAlreadySelected && ! hasReachedMaxStackCount